packages/@aws-cdk/aws-codestarnotifications/lib/target.ts

```diff
@@ -0,0 +1,28 @@
+import { INotificationRule } from './notification-rule';
```

Comment:
This file should be called `notification-rule-target.ts` instead.

Reply:
Already changed.